integration_guides/claude_code.md

```diff
@@ -6,40 +6,61 @@
 npm install -g @anthropic-ai/claude-code
 ```
 
-2. Locate where Claude Code stores its configuration, usually on Mac it is at `~/.claude.json`
-3. Edit the `mcpServers` object in the `json` file with
-
-```json
-{
-  "mcpServers": {
-    "aptos-mcp": {
-      "command": "npx",
-      "args": ["-y", "@aptos-labs/aptos-mcp"],
-      "type": "stdio",
-      "env": {
-        "APTOS_BOT_KEY": "<bot_api_key>"
-      }
-    }
-  }
-}
+2. Obtain your `APTOS_BOT_KEY`:
```

Comment:
Either we use the same steps mentioned here to generate a bot api key, or we link to this section from here - I prefer the later
https://github.com/aptos-labs/aptos-npm-mcp?tab=readme-ov-file#generate-a-build-bot-api-key